src/librustc_middle/sir.rs

```diff
@@ -114,7 +114,6 @@ impl SirFuncCx<'tcx> {
                 symbol_name,
                 blocks,
                 flags,
-                num_locals,
```

Comment:
Should we also get rid of the comment on line 107 and fold 108/109 together?

Reply:
Oops! yes!